omnibus/config/templates/datadog-agent/sysvinit_debian.erb

```diff
@@ -0,0 +1,161 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides: datadog-agent
+# Short-Description: Start and stop datadog agent
+# Description: Datadog Agent
+# Required-Start: $remote_fs
+# Required-Stop: $remote_fs
+# Should-Start: datadog-process datadog-trace
```

Comment:
any reason not to be consistent in the service names? (i.e. change `datadog-process` to `datadog-agent-process`, and same for trace)

Reply:
done